media-libs/opencolorio/opencolorio-1.1.0-r2.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Compatibility with Python 3 is declared by upstream, but it is broken in fact, check on bump
```

Comment:
is that still relevant?

Reply:
I think it's no longer relevant. A basic test in a python shell
`import PyOpenColorIO; PyOpenColorIO.GetCurrentConfig();` works with python-3.6